gtk4/src/auto/flags.rs

```diff
@@ -647,6 +647,68 @@ impl SetValue for PopoverMenuFlags {
     }
 }
 
+bitflags! {
+    pub struct PrintCapabilities: u32 {
```

Comment:
This probably wants some `Since` markers?

Reply:
They were always there, just not exposed to the Gir file, do they really need one in such case?

Reply:
Were they exported by the library in 4.0? Then this is OK

Reply:
Yes, they were exported at least in the C api since v3 at least. It just never made it way to the gir files, the bindings neither.

Reply:
Ah nice, then everything is less bad than it appeared first :)